src/test/java/com/synopsys/integration/jenkins/wrapper/JenkinsProxyHelperTest.java

```diff
@@ -126,48 +110,46 @@ private void noProxyInfoValidation(ProxyInfo proxyInfo) {
     }
 
     @ParameterizedTest
-    @MethodSource({ "validProxyInfo", "validProxyInfoWithNtlmDomain" })
-    public void testConstructorValidProxyInfo(String url, List<Pattern> ignoredProxyHosts, String testProxyUsername) {
-        ProxyInfo proxyInfo = new JenkinsProxyHelper(proxyHost, proxyPort, proxyUsername, proxyPassword, ignoredProxyHosts, testProxyUsername, ntlmWorkstation).getProxyInfo(url);
-        validProxyInfoValidation(proxyInfo, proxyHost, proxyPort, proxyUsername, proxyPassword, testProxyUsername, ntlmWorkstation);
+    @MethodSource({ "validProxyInfoInputTestData" })
+    public void testConstructorValidProxyInfo(String url, List<Pattern> ignoredProxyHosts, String testNtlmDomain) {
+        ProxyInfo proxyInfo = new JenkinsProxyHelper(expectedProxyHost, expectedProxyPort, expectedProxyUsername, expectedProxyPassword, ignoredProxyHosts, testNtlmDomain, expectedNtlmWorkstation).getProxyInfo(url);
+        validProxyInfoValidation(proxyInfo, testNtlmDomain, expectedNtlmWorkstation);
     }
 
     @ParameterizedTest
-    @MethodSource({ "validProxyInfo", "validProxyInfoWithNtlmDomain" })
-    public void testFromJenkinsValidProxyInfoJenkins(String url, List<Pattern> ignoredProxyHosts, String testNtlmDomain) {
-        String returnProxyUserName = (testNtlmDomain == null ? "" : ntlmDomain + "\\") + proxyUsername;
+    @MethodSource({ "validProxyInfoInputTestData" })
+    public void testFromJenkinsValidProxyInfoJenkins(String url, List<Pattern> ignoredProxyHosts, String expectedNtlmDomain, String proxyGetUserName) {
         ProxyConfiguration proxyConfigurationMock = Mockito.mock(ProxyConfiguration.class);
         Jenkins jenkinsMock = Mockito.mock(Jenkins.class);
         JenkinsWrapper jenkinsWrapper = new JenkinsWrapper(jenkinsMock);
 
         setJenkinsFields(jenkinsMock, proxyConfigurationMock);
 
-        Mockito.when(proxyConfigurationMock.getUserName()).thenReturn(returnProxyUserName);
-        Mockito.when(proxyConfigurationMock.getPassword()).thenReturn(proxyPassword);
+        Mockito.when(proxyConfigurationMock.getUserName()).thenReturn(proxyGetUserName);
+        Mockito.when(proxyConfigurationMock.getPassword()).thenReturn(expectedProxyPassword);
         Mockito.when(proxyConfigurationMock.getNoProxyHostPatterns()).thenReturn(ignoredProxyHosts);
 
         ProxyInfo proxyInfo = JenkinsProxyHelper.fromJenkins(jenkinsWrapper).getProxyInfo(url);
-        validProxyInfoValidation(proxyInfo, proxyHost, proxyPort, proxyUsername, proxyPassword, testNtlmDomain, StringUtils.EMPTY);
+        validProxyInfoValidation(proxyInfo, expectedNtlmDomain, StringUtils.EMPTY);
     }
 
-    private void validProxyInfoValidation(ProxyInfo proxyInfo, String proxyHost, int proxyPort, String proxyUsername, String proxyPassword,
-        String ntlmDomain, String ntlmWorkstation) {
+    private void validProxyInfoValidation(ProxyInfo proxyInfo, String ntlmDomain, String ntlmWorkstation) {
```

Comment:
Since 4 of the input params used for verification were always using the static variables, these were removed from the method signature and the static variables were used directly.